src/grt/src/heatMapRudy.cpp

```diff
@@ -66,7 +66,12 @@ void RUDYDataSource::populateXYGrid()
     return;
   }
 
-  rudy_ = grouter_->getRudy();
+  try {
+    rudy_ = grouter_->getRudy();
+  } catch (...) {
+    gui::GlobalRoutingDataSource::populateXYGrid();
```

Comment:
@maliberty Silently `catch (...)` Shudder.... logging?

Reply:
The message is issued when error () is called

Reply:
Those exceptions have a base class, right?

Catch only those exceptions...

Reply:
resolved in #5129 5129